tests/state/ViewerState/constants.ts

```diff
@@ -1,15 +1,51 @@
 import { Dataset } from "../../../src/colorizer";
+import { AnyManifestFile } from "../../../src/colorizer/utils/dataset_utils";
+import { makeMockDataset } from "../../test_utils";
 
 export const DEFAULT_BACKDROP_KEY = "test_backdrop_key";
 
-// TODO: This will likely need to be an actual dataset object at some point
-// as changing the dataset has more side effects.
```

Comment:
The TODO in question. The follow-up PR to this introduces feature keys, which requires more Dataset functionality. Mocking datasets makes the unit tests more robust.